exercises/scripts/worker.sh

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env bash
 
 NODE=$1
-NODE_HOST_IP=20+$NODE
+NODE_HOST_IP="192.168.56.$((20+$NODE))"
 
 $(cat /vagrant/kubeadm-init.out | grep -A 2 "kubeadm join" | sed -e 's/^[ \t]*//' | tr '\n' ' ' | sed -e 's/ \\ / /g')
 
+echo "KUBELET_EXTRA_ARGS=--node-ip=$NODE_HOST_IP --cgroup-driver=systemd" > /etc/default/kubelet
```

Comment:
This is the recommended way to pass extra args within kubernetes systemd ExecStart. Because if you update kubelet, there are chances that the systemd file will be overriden and not the `/etc/default/kubelet` file. Not very useful in this situation, but it worth using best practices.

Reply:
Thanks for the info. Very helpful.